src/components/CreateWallet.jsx

```diff
@@ -8,6 +8,35 @@ import ToggleSwitch from './ToggleSwitch'
 import { serialize, walletDisplayName } from '../utils'
 import { useServiceInfo } from '../context/ServiceInfoContext'
 import * as Api from '../libs/JmWalletApi'
+import './CreateWallet.css'
```

Comment:
This is a bit of an anti-pattern I believe. Using separate files to scope CSS is fine but it should be [CSS Modules](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/) then. CSS Modules let you use the same CSS class name in different files without worrying about naming clashes. Having to worry about unique naming across several separate files can become cumbersome quickly.

Reply:
Ah.. thank you. Really thought this is a good idea when only styles specific to the current screens are placed inside own files (and tightly scoped with `.<screen_name> <class>`). I will stop doing this and revert these changes in upcoming commits. Thanks for your feedback. :orange_heart:

Reply:
We can definitely start splitting CSS up into separate files -- your point about styles which are specific to single components is totally valid. My only complaint is that if we do it, it should probably be done using CSS Modules. :)